packages/replay-internal/test.setup.ts

```diff
@@ -1,17 +1,17 @@
-import { TextEncoder } from 'util';
+import { vi } from 'vitest';
+import type { Assertion, AsymmetricMatchersContaining, Mocked, MockedFunction } from 'vitest';
```

Comment:
Wasn't sure what people would prefer, globals vs explicit imports. For this PR I'm just importing `vi` and types, but can follow-up to make it consistent.